tests/console/ntp_client.pm

```diff
@@ -14,43 +14,52 @@ use strict;
 use warnings;
 use testapi;
 use utils;
-use version_utils 'is_sle';
+use version_utils qw(is_sle is_jeos);
 
 sub run {
     my $self = shift;
     $self->select_serial_terminal;
 
     assert_script_run 'timedatectl';
 
-    if (is_sle) {
+    if (is_sle('>15')) {
         systemctl 'enable chronyd';
         systemctl 'start chronyd';
         # bsc#1179022 avoid '503 No such source' error while chrony does pick responding sources after start
         script_run qq(timeout 85 bash -c 'until chronyc activity|grep "0 sources doing burst.*online"; do sleep 1; echo "waiting for ntp sources response"; done' -k);
     }
 
     # ensure that ntpd is neither installed nor enabled nor active
-    systemctl 'is-active ntpd',  expect_false => 1;
-    systemctl 'is-enabled ntpd', expect_false => 1;
-    if (script_run('rpm -q ntp') == 0) {
-        # ntp should not be installed by default as we are using chrony
-        record_soft_failure 'boo#1114189 - ntp is installed by default';
+    if (is_jeos && is_sle('=12-sp5')) {
```

Comment:
ntpd is tested in `ntp.pm`, please do not extend this module.

Reply:
move to yaml replacing ntp_client